nathan-shulkin.html

```diff
@@ -8,15 +8,15 @@
   <body>
     <h1>Brief History of CSS</h1>
     <ul>
-      <li><div id="green" class="chrono">1994</div>: The first work on style sheets started at CERN.</div></li>
+      <li><span id="green" class="chrono">1994</span>: The first work on style sheets started at CERN.</div></li>
       <li><div class="chrono">December 1996</div>: CSS was proposed as W3C standard:
```

Comment:
Get rid of this `<div>` and replace with a `<span>` as you've done for the other dates.